src/token_cache.rs

```diff
@@ -71,10 +72,21 @@ struct TokenCacheValue {
     expiration: u64,
 }
 
-#[derive(Default, Clone)]
+#[derive(Clone)]
 pub(crate) struct TokenCache {
     // (registry, repository, scope) -> (token, expiration)
     tokens: Arc<RwLock<BTreeMap<TokenCacheKey, TokenCacheValue>>>,
+    /// Default token expiration in seconds, to use when claim doesn't specify a value
+    pub default_expiration_secs: usize,
+}
+
+impl Default for TokenCache {
```

Comment:
Sorry, I should have pushed as a separate commit.  This was brought in in the last amended commit.

1. Does this make sense to keep/use if we bring back `new(...)`?
  a. If we keep this, is `Arc::default()` the right default for `tokens`?